news/2024/2024-07-25-introducing-daily-challenges.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: Introducing Daily Challenges
+date: 2024-07-25 01:30:00 +0000
+---
+
+A new way to play the game!
+
+![](/wiki/shared/news/2024-07-25-introducing-daily-challenges/banner.jpg)
+
+By popular demand, osu! will be taking a page from the books of other games. Watch [pishifat](https://osu.ppy.sh/users/3178418) explain what's happening along with [peppy](https://osu.ppy.sh/users/2) in the video below, or read on for the quick summary.
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/e8tKtedQvYc" frameborder="0" allowfullscreen></iframe>
+</div>
+
+---
+
+![](/wiki/shared/news/2024-07-25-introducing-daily-challenges/cup-of-the-day-comment.jpg)
+
+With support from almost 3,000 of you in the comments of [a previous YouTube video](https://www.youtube.com/watch?v=xUSxEjQQ1UI), **daily challenges** are now live!
+
+These are daily featured beatmaps accessible from the "play" section of the main menu, where anyone can compete with live updates.
+
+![](/wiki/shared/news/2024-07-25-introducing-daily-challenges/daily-challenge-screen.jpg)
+
+Each day will introduce one new map with increased difficulty, resetting every Monday.
+
+![](/wiki/shared/news/2024-07-25-introducing-daily-challenges/daily-challenge-calendar.png?1)
+
+After passing the beatmap of the day, it will be counted towards "daily challenge" totals on the profile (which are updated once per day).
```

Comment:
```suggestion
After passing the beatmap of the day, it will be counted towards "daily challenge" totals on the profile (which are updated at the end of each day).
```